ruby/README.md

```diff
@@ -3,19 +3,6 @@
 This folder contains example Ice applications written in Ruby. Each example is a simple client application that
 demonstrates a specific feature or programming technique.
 
-> Ice for Ruby is supported on Linux and macOS platforms.
```

Comment:
I agree that listing "MATLAB R2024a" is too far, but I think this kind of 'supported platforms' sentence should be moved to `NIGHTLY.md` instead of completely deleted.

It definitely doesn't need to live in this README, but I think it would make sense to have in
[NIGHTLY.md#ice-for-ruby](https://github.com/zeroc-ice/ice/blob/main/NIGHTLY.md#ice-for-ruby)
Otherwise it's not obvious to users that this isn't supported on Windows.